Cargo.toml

```diff
@@ -63,7 +63,7 @@ all = "warn"
 
 [workspace.dependencies]
 # stylus-related
-stylus-sdk = { version = "=0.6.0", default-features = false }
+stylus-sdk = "0.6.0"
 stylus-proc = { version = "=0.6.0", default-features = false }
```

Comment:
Should we remove `default-features` from `stylus-proc` and `alloy` too?

Reply:
In this case I think we should "cherry pick" the commit, that uses `stylus-proc` indirectly through `stylus-sdk` from the `main` branch. I hope we will get less merge conflicts

Reply:
I don't think for alloy we want to do that, since it includes std by [default](https://docs.rs/crate/alloy/latest/features)